webpack.config.js

```diff
@@ -14,6 +14,11 @@ var config = {
     },
     module: {
         loaders: [
+            {
+                test: /\.js$/,
+                use: ["source-map-loader"],
+                enforce: "pre"
+            },
```

Comment:
Allows Webpack to include the original source code for `ultra-responsive-calendar` in its sourcemap to aid in debugging.